engine/src/main/java/com/google/android/fhir/sync/upload/request/IndividualRequestGenerator.kt

```diff
@@ -20,21 +20,20 @@ import ca.uhn.fhir.context.FhirContext
 import ca.uhn.fhir.context.FhirVersionEnum
 import com.google.android.fhir.ContentTypes
 import com.google.android.fhir.LocalChange
-import com.google.android.fhir.LocalChange.Type
-import com.google.android.fhir.db.impl.dao.LocalChangeToken
 import com.google.android.fhir.sync.UrlUploadRequest
+import com.google.android.fhir.sync.upload.patch.Patch
 import org.hl7.fhir.r4.model.Binary
 import org.hl7.fhir.r4.model.Resource
 import org.hl7.fhir.r4.model.codesystems.HttpVerb
 
 /** Generates list of [UrlUploadRequest]s with for each [LocalChange] given. */
 class IndividualRequestGenerator(
   private val getIndividualRequestForLocalChangeType:
```

Comment:
update function name to reflect we are dealing with Patches
```suggestion
  private val getIndividualRequestForPatchType:
```